<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>741</width>
                <height>626</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QGraphicsView" name="graphicsView">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>160</y>
                        <width>741</width>
                        <height>441</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">background-color:white;

</string>
                </property>
            </widget>
            <widget class="QLabel" name="label">
                <property name="geometry">
                    <rect>
                        <x>190</x>
                        <y>170</y>
                        <width>391</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Verdana</family>
                        <pointsize>11</pointsize>
                        <weight>50</weight>
                        <bold>false</bold>
                    </font>
                </property>
                <property name="styleSheet">
                    <string notr="true">color:black;
background-color: white;
border: 2px solid white; border-radius:10px;</string>
                </property>
                <property name="text">
                    <string>Evaluación de conocimientos: Has la siguiente letra</string>
                </property>
            </widget>
            <widget class="QTextBrowser" name="textBrowser">
                <property name="enabled">
                    <bool>true</bool>
                </property>
                <property name="geometry">
                    <rect>
                        <x>380</x>
                        <y>210</y>
                        <width>331</width>
                        <height>271</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">
border: 5px solid #eb4034;
background-color:black;
font-size: 40px;
color:white;
</string>
                </property>
                <property name="frameShape">
                    <enum>QFrame::Box</enum>
                </property>
                <property name="html">
                    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:40px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_2">
                <property name="geometry">
                    <rect>
                        <x>470</x>
                        <y>500</y>
                        <width>160</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Times New Roman</family>
                        <pointsize>16</pointsize>
                        <weight>50</weight>
                        <italic>false</italic>
                        <bold>false</bold>
                    </font>
                </property>
                <property name="styleSheet">
                    <string notr="true">QPushButton#pushButton_2{
color: white;
background-color:black;
border-radius:10px;
font: 16pt &quot;Times New Roman&quot;;
background-repeat:none;
}
QPushButton#pushButton_2:hover
{
	background-color:#eb4034;
	color:black;
}</string>
                </property>
                <property name="text">
                    <string>Detener cámaras</string>
                </property>
            </widget>
            <widget class="QGraphicsView" name="graphicsView_3">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>741</width>
                        <height>161</height>
                    </rect>
                </property>
                <property name="toolTip">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="styleSheet">
                    <string notr="true">background-color: #eb4034;
</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_2">
                <property name="geometry">
                    <rect>
                        <x>50</x>
                        <y>0</y>
                        <width>631</width>
                        <height>70</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Verdana</family>
                        <pointsize>18</pointsize>
                        <weight>50</weight>
                        <bold>false</bold>
                    </font>
                </property>
                <property name="styleSheet">
                    <string notr="true">color:white;</string>
                </property>
                <property name="text">
                    <string>Reconocimiento del abecedario en lenguaje de señas</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_3">
                <property name="geometry">
                    <rect>
                        <x>40</x>
                        <y>210</y>
                        <width>321</width>
                        <height>271</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">border: 5px solid #eb4034;
font-size: 40px;
color:white;
</string>
                </property>
                <property name="text">
                    <string>TextLabel</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_5">
                <property name="geometry">
                    <rect>
                        <x>31</x>
                        <y>490</y>
                        <width>331</width>
                        <height>51</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">
font-size: 40px;
color:white;
</string>
                </property>
                <property name="text">
                    <string>TextLabel</string>
                </property>
            </widget>
            <widget class="QSlider" name="trackbar">
                <property name="geometry">
                    <rect>
                        <x>50</x>
                        <y>500</y>
                        <width>301</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">color: rgb(0, 0, 0);</string>
                </property>
                <property name="maximum">
                    <number>255</number>
                </property>
                <property name="singleStep">
                    <number>1</number>
                </property>
                <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedAppearance">
                    <bool>false</bool>
                </property>
                <property name="invertedControls">
                    <bool>false</bool>
                </property>
            </widget>
            <widget class="QTextBrowser" name="textBrowser_3">
                <property name="enabled">
                    <bool>true</bool>
                </property>
                <property name="geometry">
                    <rect>
                        <x>570</x>
                        <y>210</y>
                        <width>141</width>
                        <height>131</height>
                    </rect>
                </property>
                <property name="styleSheet">
                    <string notr="true">border: 5px solid #eb4034; border-radius:20px;
background-color:black;</string>
                </property>
                <property name="frameShape">
                    <enum>QFrame::Box</enum>
                </property>
                <property name="html">
                    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
            </widget>
            <widget class="QCommandLinkButton" name="linkButton">
                <property name="geometry">
                    <rect>
                        <x>700</x>
                        <y>170</y>
                        <width>31</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="text">
                    <string>CommandLinkButton</string>
                </property>
            </widget>
            <widget class="QPushButton" name="scan_sinlge">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>80</y>
                        <width>250</width>
                        <height>61</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Times New Roman</family>
                        <pointsize>11</pointsize>
                        <weight>50</weight>
                        <italic>false</italic>
                        <bold>false</bold>
                    </font>
                </property>
                <property name="layoutDirection">
                    <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                    <bool>false</bool>
                </property>
                <property name="styleSheet">
                    <string notr="true">QPushButton#scan_sinlge{
padding-left:-40px;
color:white;
background-color:black;
border: 2px solid #f00c0c; border-radius:30px;
font: 11pt &quot;Times New Roman&quot;;
background-image:url(&quot;icons/scan.png&quot;);
background-repeat:none;
}
QPushButton#scan_sinlge:hover
{
    background-color:#f00c0c;
    color:black;
}</string>
                </property>
                <property name="text">
                    <string>                          Comenzar escaneo</string>
                </property>
            </widget>
            <widget class="QPushButton" name="scan_test">
                <property name="geometry">
                    <rect>
                        <x>300</x>
                        <y>80</y>
                        <width>250</width>
                        <height>61</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Times New Roman</family>
                        <pointsize>11</pointsize>
                        <weight>50</weight>
                        <italic>false</italic>
                        <bold>false</bold>
                    </font>
                </property>
                <property name="layoutDirection">
                    <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                    <bool>false</bool>
                </property>
                <property name="styleSheet">
                    <string notr="true">QPushButton#scan_test{
    padding-left:-40px;
    color:white;
    background-color:black;
    border: 2px solid #f00c0c; border-radius:30px;
    font: 11pt &quot;Times New Roman&quot;;
    background-image:url(&quot;icons/scan.png&quot;);
    background-repeat:none;
    }
    QPushButton#scan_test:hover
    {
    background-color:#f00c0c;
    color:black;
    }</string>
                </property>
                <property name="text">
                    <string>                          Comenzar prueba</string>
                </property>
            </widget>
            <widget class="QPushButton" name="go_menu">
                <property name="geometry">
                    <rect>
                        <x>630</x>
                        <y>80</y>
                        <width>100</width>
                        <height>61</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Times New Roman</family>
                        <pointsize>11</pointsize>
                        <weight>50</weight>
                        <italic>false</italic>
                        <bold>false</bold>
                    </font>
                </property>
                <property name="layoutDirection">
                    <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                    <bool>false</bool>
                </property>
                <property name="styleSheet">
                    <string notr="true">QPushButton#go_menu{
    padding-left:-40px;
    color:white;
    background-color:black;
    border: 2px solid #f00c0c; border-radius:30px;
    font: 11pt &quot;Times New Roman&quot;;
    background-repeat:none;
    }
    QPushButton#go_menu:hover
    {
    background-color:#f00c0c;
    color:black;
    }</string>
                </property>
                <property name="text">
                    <string>          Menu</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_4">
                <property name="geometry">
                    <rect>
                        <x>50</x>
                        <y>540</y>
                        <width>311</width>
                        <height>16</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Verdana</family>
                    </font>
                </property>
                <property name="text">
                    <string>Descripción: Selector de brillo ajustado por un slider.</string>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>741</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar" />
    </widget>
    <resources />
    <connections />
</ui>
